source/docs/api-reference/api-usage/actuator-limits.rst

```diff
@@ -33,3 +33,15 @@ The state of the forward or reverse limit switch can be retrieved from the API v
          if (forwardLimit.GetValue() == signals::ForwardLimitValue::ClosedToGround) {
             // do action when forward limit is closed
          }
+
+   .. tab-item:: Python
+      :sync: Python
+
+      .. code-block:: python
+
+         import phoenix6
```

Comment:
FYI, there's going to be changes to this in beta-5. Python is going to follow the same submodule structure as C++. So in this case, we would need:
```py
from phoenix6 import signals.
# ...
if forward_limit.value is signals.ForwardLimitValue.CLOSED_TO_GROUND:
```
Also, all our Python examples use `snake_case`, we don't need `m_`, and if statements don't need parentheses. (see https://github.com/CrossTheRoadElec/Phoenix6-Examples/blob/python_examples/python/ArcadeDrive/robot.py)

Reply:
In that case, does C++ also need to get refactored to follow C++ conventions?